.changeset/khaki-brooms-sort.md

```diff
@@ -0,0 +1,5 @@
+---
+'@emotion/react': patch
+---
+
+Remove hasOwnProperty from utils
```

Comment:
Renamed an internal `hasOwnProperty` to `hasOwn`. This avoids problems in CommonJS environments when the consumer tries to prevent prototype pollution with `Object.freeze(Object.prototype)`.

Reply:
eh, this was meant to be a code suggestion and not a regular comment 😅